src/main/java/uk/gov/pay/publicauth/util/DependentResourceWaitCommand.java

```diff
@@ -3,6 +3,8 @@
 import io.dropwizard.cli.ConfiguredCommand;
 import io.dropwizard.setup.Bootstrap;
 import net.sourceforge.argparse4j.inf.Namespace;
+import uk.gov.pay.commons.utils.startup.ApplicationStartupDependentResourceChecker;
```

Comment:
I have just realised that we could have a constructor of this which does not accept a waiter, and instantiates it internally. This would avoid having to duplicate the waiter instantiation in each microservice, too. Could be an improvement for the second pass :)

Reply:
You have commit access to the code, @simad  :)